pkg/cmd/repo/edit/edit_test.go

```diff
@@ -250,7 +405,7 @@ func Test_editRun_interactive(t *testing.T) {
 			promptStubs: func(pm *prompter.MockPrompter) {
 				pm.RegisterMultiSelect("What do you want to edit?", nil, editList,
 					func(_ string, _, opts []string) ([]int, error) {
-						return []int{0, 2, 3, 5, 6, 8, 9}, nil
+						return []int{0, 2, 3, 5, 6, 9}, nil
```

Comment:
That's a yikes (not your fault)

Reply:
I suspect the tight coupling of this code versus the code under test and the ordering of options is 🤢 which I agree isn't ideal.